### ISSUE TITLE
Linearization of inheritance graph impossible as of v1.11.0

### ISSUE THREAD

**Issue by mswezey23** [OpenZeppelin/openzeppelin-solidity#1110]
## 🎉 Description
- [X ] 🐛 This is a bug report.

## 💻 Environment
OZ version: v1.10.0 and v1.11.0
Network: Ganache
Tooling: Truffle

## 📝 Details

`TypeError: Linearization of inheritance graph impossible as of v1.10.0`
The contracts: `WhitelistedCrowdsale` & `FinalizableCrowdsale` with `Ownable` being the cause.

## 🔢 Code To Reproduce Issue [ Good To Have ]
```
CustomCrowdsale is CappedCrowdsale, MintedCrowdsale, WhitelistedCrowdsale, FinalizableCrowdsale
```

## 👍 Other Information
This was working okay in v1.9.0. `WhitelistedCrowdsale` was update in v1.10.0 which introduced this 🐛.

**Comment by frangio** [comment]
Thanks for reporting @mswezey23! I was able to reproduce this in 1.11.0, but not in 1.10.0. My understanding is that the bug was introduced in #981, which was only included in 1.11.0.

I think the problem is caused by this line:
https://github.com/OpenZeppelin/openzeppelin-solidity/blob/73be06412fdeb0453c11fa708d4e211947569ec3/contracts/crowdsale/distribution/FinalizableCrowdsale.sol#L13

Swapping `Ownable` and `TimedCrowdsale` makes linearization possible.

It should've been like that from the beginning because of the rule that "most base-like" goes to the left of the inheritance list.

**Comment by mswezey23** [comment]
Ah, that could have been my mix up. Yarn and I were having some issues & we had to duke it out. Apparently manually updating the versions in the `package.json` file to downgrade the OZ version doesn't work. It keeps the latest in the `yarn.lock` file. Had to run the commands `yarn remove ...` and `yarn add ...` for a successful downgraded version... back to topic now.

Thanks, @frangio, for the additional insight and confirming the bug! 

Glad the fix was easy as swapping the order 😎
